packages/discord.js/src/client/Client.js

```diff
@@ -253,6 +253,23 @@ class Client extends BaseClient {
     this.rest.setToken(null);
   }
 
+  /**
+   * Options used when deleting a webhook from Discord.
```

Comment:
The "from Discord" part is superfluous. Everything in this library is for Discord. No need to state "Create an event in Discord", "Delete a channel from Discord", etc.

```suggestion
   * Options used for deleting a webhook.
```

Reply:
Well every jsdoc here uses that
But yeah i agree

https://github.com/discordjs/discord.js/blob/5de608a576a0b390e45074bed16b8dd47710b38e/packages/discord.js/src/client/Client.js#L281

Reply:
We should move away from those patterns. I mean, that's what this library is for, after all.

Reply:
that's probably to clarify that it does an API request rather than using cache